README.md

```diff
@@ -43,8 +42,7 @@ Getting help
 ------------
 
 If you get stuck, have a question, or want to clarify any aspect of the
-homework, you can contact members of our development team by
-joining our [public Slack channel.](https://public-slack.adhoc.team)
+homework, use the chat button in the lower right to contact us.
```

Comment:
Since people will only be looking at this version of the readme on github, should we not say "use the chat button in the lower right" but "use the chat button on homework.adhoc.team ?

Reply:
:+1: